src/renderer/components/+workloads-cronjobs/cronjob.store.ts

```diff
@@ -34,7 +34,7 @@ export class CronJobStore extends KubeObjectStore<CronJob> {
   }
 
   getStatuses(cronJobs?: CronJob[]) {
-    const status = { suspended: 0, scheduled: 0 };
```

Comment:
Why does the order matter?

Reply:
This is an object, so I think we should specify the order by extracting to an array. Or at least add a comment above all of these saying that the order is important.

Reply:
Since we're decided to put all "green" statuses on top (show them first), I've moved `scheduled` to be first one since it's "green".

Reply:
I still think that should be something that is commented or specified somewhere else. Since objects are key/value structures which don't really have a conceptual ordering.

Reply:
Yes, objects didn't have conceptual ordering, but they're actually have an order for now. The results are always the same + we have tests now checking ordering.

Here's mentions about object ordering https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#objects_vs._maps

As an alternative, I've converted statuses to use Maps, but this looks more verbose without visible benefits.

Reply:
Okay, I was thinking more of having a defined ordering in `overview-workload-status.tsx` but that might be a bit too much. Maybe something for the future.

I don't particularly like the fact that this ordering is required to be static but I concur that should be for the future.